### DISCUSSION TITLE
CU/DU on K8s in Split8 model with ZMQ simulated Radio

### DISCUSSION THREAD

**Discussion by jmarkotic** [srsran/srsRAN_Project#562] [General Help] [answered]
Hi,
first, kudos to great advancements on project (i came back to learn after some time just o see great progress).
I was reading tutorial on CU/DU ORAN 7.2 On Kuberneets cluster but i was looking for more home/lab friendly setup (no O-RU).
Per my understanding, split 8 is supported as well (both low/high PHY processed by DU) and one could leverage USDP/UHD. But I guess also ZMQ should also work in split8 model ?

Are any of docker images provided with zmq support compiled ? (i guess I could build it myself).
For what i understand i could use existing provided helm charts for 7.2 example and just replace gNB config with ZMQ setup for monolith gNB (since it is still monolith cu/du inside single container) ?

**Comment by pgawlowicz** [comment, marked as answer]
We provide two tutorials demonstrating how to send samples over ZMQ in split 8:
- [with srsUE](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#zeromq-based-setup)
- [with AmarisoftUE](https://docs.srsran.com/projects/project/en/latest/tutorials/source/amariUE/source/index.html#zeromq-based-setup)

**Comment by jmarkotic** [comment]
Thanks for answer&info provided. I will use that gNB config in K8s/container setup. As for docker image, i quickly checked dockerfile, and i believe zmq is compiled in (and if not i think can be added).
